python/core/auto_generated/qgsreadwritecontext.sip.in

```diff
@@ -111,6 +111,24 @@ Sets the project translator. Means it shouldn't conform mDefaultTranslator anymo
 It's usually the QgsProject where the function with the context is made and won't be changed anymore.
 
 .. versionadded:: 3.4
+%End
+
+    QgsCoordinateTransformContext transformContext() const;
```

Comment:
Are these additions still necessary? I gather they aren't currently used here

Reply:
`transformContext` it's used in `readXml`, and getting it from the rw context seems the natural way to do it.